frontend/app/views/fragments/event/listing.scala.html

```diff
@@ -68,34 +68,30 @@ <h3 class="listing__title h-intro">
         </section>
     }
 
-    @if(showPastEvents && eventPortfolio.pastEvents) {
```

Comment:
quite surprised that used to compile... `eventPortfolio.pastEvents` is an Option behaving like a boolean? Wonder if play templates is getting involved there.